# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../cineon/install.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- cineon/Makefile_orig	2006-08-26 18:14:01.000000000 +0100
+++ cineon/Makefile	2006-08-26 18:17:57.000000000 +0100
@@ -11,7 +11,9 @@
 
 cineon: cineon.c $(SOURCE) $(INCLUDE)
 	cc -O2 -Wall -o cineon cineon.c $(SOURCE) $(GIMPARGS)
-	gimptool --install-bin cineon
+
+install: cineon
+	gimptool-2.0 --install-admin-bin cineon
 
 cineonTest: cineonTest.c $(SOURCE) $(INCLUDE)
 	cc -O2 -g -o cineonTest cineonTest.c $(SOURCE) -lm
